src/coreclr/tools/dotnet-pgo/Program.cs

```diff
@@ -276,7 +276,7 @@ static int InnerDumpMain(CommandLineOptions commandLineOptions)
 
             PrintDetailedMessage($"Parsing {commandLineOptions.InputFileToDump}");
             var profileData = MIbcProfileParser.ParseMIbcFile(tsc, mibcPeReader, null, onlyDefinedInAssembly: null);
-            PrintMibcStats(profileData);
+            PrintMibcStats(ParseMibcConfig(tsc, mibcPeReader), profileData);
```

Comment:
Would it be better to add the config information to `ProfileData` itself?

Reply:
moved